tests/h/services/list_organizations_test.py

```diff
@@ -42,44 +60,3 @@ def test_provides_request_db_as_session(self, pyramid_request):
         svc = list_organizations_factory(None, pyramid_request)
 
         assert svc._session == pyramid_request.db
-
-
-@pytest.fixture
-def authority(pyramid_request):
-    return pyramid_request.default_authority
-
-
-@pytest.fixture
-def alternate_authority():
-    return "bar.com"
-
-
-@pytest.fixture
-def org_svc(pyramid_request):
-    return organization_factory(None, pyramid_request)
```

Comment:
Also took the time to remove this oddity in order to work out what was going on. This creates a real example of the service to generate it's own fixtures, which isn't needed as a factory object will do the same things perfectly well.

Almost all of these fixtures are actually specific to a particular test or other / don't need to exist.